schema/campaign_spec.schema.json

```
@@ -85,6 +85,30 @@
         }
       }
     },
+    "importChangesets": {
+      "type": "array",
+      "description": "Import existing changesets on code hosts.",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["repository", "externalIDs"],
+        "properties": {
+          "repository": {
+            "type": "string",
+            "description": "The repository name as configured on your Sourcegraph instance."
+          },
+          "externalIDs": {
+            "type": "array",
+            "description": "The changesets to import from the code host. For GitHub this is the PR number, for Gitlab this is the MR number, for Bitbucket server this is the PR number.",
```

```suggestion
            "description": "The changesets to import from the code host. For GitHub this is the PR number, for GitLab this is the MR number, for Bitbucket Server this is the PR number.",
```

I wish everyone would pick a capitalisation scheme and stick to it. :smiley:

---

🤦 haha

Thanks!